tests/TestOne/buildtool/bin/buildtool

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+#
+# This is a very simple pjc-aware ocamlfind-based build tool.
+
+# Build
+
+DEPENDS=""
+for p in $cur_depends; do
```

Comment:
I think this should be `$cur__depends` (double underscore to signify that it's part of the ecosystem - not any specific build script). (I commented elsewhere, where the corresponding fix would go).